ci/check_migration_files.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+set -euo pipefail
+# This file checks for the existence of downgrade scripts for every upgrade script that is changed in the branch.
+
+# list of migration files, and early exit if no migration scripts are touched
+migration_files=$(git diff --name-only origin/master | grep "src/backend/distributed/sql/citus--.*sql") || exit 0
```

Comment:
the `exit 0` will also hide errors from `git diff` itself, not just from `grep`. You can solve that like this:
```suggestion
migration_files=$(git diff --name-only origin/master | { grep "src/backend/distributed/sql/citus--.*sql" || exit 0 ; })
```